src/interface/src/app/plan/delete-note-dialog/delete-note-dialog.component.scss

```diff
@@ -0,0 +1,31 @@
+.delete-note-dialog {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    padding: 24px;
+}
+
+.dialog-heading {
+    text-align: left;
+}
+
+.warning-icon {
+    color: #f3b300;
+    font-size: 60px;
+    height: 60px;
+    width: 60px;
+}
+
+p {
+    font-family: 'Roboto';
```

Comment:
I dont think we want roboto, is this intentional?
Also- you might already have the styles for this as a mixin

Reply:
Ha, wow, no --  I think I started from the styling in DeleteAccountDialog, and didn't even question it. I'll fix...